src/main/java/org/circle8/service/PuntoReciclajeService.java

```diff
@@ -31,4 +36,83 @@ public List<PuntoReciclajeDto> list(PuntoReciclajeFilter filter) throws ServiceE
 			throw new ServiceError("Ha ocurrido un error al obtener el listado de puntos de reciclaje", e);
 		}
 	}
+
+	/**
+	 * Guarda un nuevo punto de reciclaje
+	 * @param dto
+	 * @return
+	 * @throws ServiceError
+	 */
+	public PuntoReciclajeDto save(PuntoReciclajeDto dto) throws ServiceError {
+		var entity = dto.toEntity();
+		try (var t = dao.open()) {
+			entity = dao.save(t, entity);
+			dto.id = entity.id;
+			for(TipoResiduoDto tr : dto.tipoResiduo){
+				dao.saveRelacion(t, tr.id, dto.id);
+			}
+			t.commit();
+		} catch (PersistenceException e) {
+			throw new ServiceError("Ha ocurrido un error al guardar el punto de reciclaje", e);
+		}
+		return dto;
+	}
+
+	/**
+	 * Obtiene un punto de reciclaje por medio de su id
+	 */
+	public PuntoReciclajeDto get(Long id, Long recicladorId) throws ServiceError, NotFoundException {
+		try {
+			return PuntoReciclajeDto.from(this.dao.get(id, recicladorId).
+				orElseThrow(() -> new NotFoundException("No existe el punto de reciclaje")));
+		} catch (PersistenceException e) {
+			throw new ServiceError("Ha ocurrido un error al obtener el listado de puntos de reciclaje", e);
+		}
+
+	}
+
+	/**
+	 * Elimina un punto de reciclaje
+	 * @param id
+	 * @param recicladorId
+	 * @return
+	 * @throws ServiceError
+	 */
+	public boolean delete(Long id, Long recicladorId) throws ServiceError {
```

Comment:
No deberia devolver boolean. No le pregunte si puede o no borrar, sino que directamente le ordeno a la funcion que lo haga.
Si quiero decir que puede ser que no lo borré por que no lo encontré, en ese caso se puede usar la exception de not found que es más acorde.